### ISSUE TITLE
Missing Form Label Error in WAVE Accessibility Tool for Reblog Form

### ISSUE THREAD

**Issue by scosgro** [Automattic/wp-calypso#46105]
#### Steps to reproduce

1. https://wave.webaim.org/report#/https://ps130brooklyn.com/
2. There is a missing form label error referencing a hidden reblog carousel https://snipboard.io/nECUkL.jpg when it should be labeled.
![image](https://user-images.githubusercontent.com/30731356/94853053-a40f7980-03df-11eb-922a-bd1f2fe042f1.png)
3. When styles are turned off, that's when you can see the form that's reporting as having a missing label.
![image](https://user-images.githubusercontent.com/30731356/94853095-b5f11c80-03df-11eb-9269-27c47377e3f6.png)

I'm not sure if this is an issue with WAVE, or what specifically is going on here. We have an open P2 p4vt7e-6Q-p2 on this specific issue, so wanted to move here to Github. 

#### What I expected
I'm not well versed here so I'm not 100% sure, but possibly that the hidden element that has the label already would not show up in WAVE as having a missing label.

#### What happened instead
WAVE gave the page a Missing Form Label error on that hidden form element (that should already have a label). 

#### Browser / OS version
All

#### Context / Source
#user-report

**Comment by sandymcfadden** [comment]
Just noting the source of the report 3315379-zd

**Comment by obenland** [comment]
This seems to have been fixed in D52789-code.
